docs/troubleshooting.md

```diff
@@ -21,7 +21,7 @@ Go to each downstream cluster and run the following command for the local cluste
 # Downstream cluster
 $ kubectl logs -f $fleet-agent-pod-name -n cattle-fleet-system
 # Local cluster
-$ kubectl logs -f $fleet-agent-pod-name -n cattle-local-fleet-system
+$ kubectl logs -f $fleet-agent-pod-name -n cattle-local-cattle-fleet-system
```

Comment:
This seems wrong? Not sure if the old namespace was a good choice either.
"cattle-fleet-system" should be right. 

While we are at it, I prefer `kubectl logs -l app=fleet-agent -n cattle-fleet-system`
 instead of letting the user guess the pod name.

Reply:
Good catch, it was a search and replace issue, thanks.